index.html

```diff
@@ -31,7 +30,126 @@ <h1 class="black-text-header">Hey there. I’m<br> Jorge<br><span>I’m a softwa
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+      <div class="recent-work six">
+        <div class="multi-post post-six">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <h2 class="about-me-header">About me</h2>
+    <p class="about-me-paragraph-mobile"> I can help you build a product,<br>feature or website Look through<br>some of
+      my work and
+      experience! If<br>you like what you see and have a<br>project you need coded,<br>don’t hesitate to contact me.</p>
+    <button class="about-me-button">Get My Resume</button>
+    <section class="about-me">
+      <div class="about-me-languages">
+        <img class="languages-icon" src="./assets/images/languages.svg">
+        <h3 class="languages-header">Languages</h3>
+        <ul class="languages-list">
+          <li class="language-element">JavaScript</li>
+          <li class="language-element">Ruby</li>
+          <li class="language-element">Html</li>
+          <li class="language-element">CSS</li>
+        </ul>
+      </div>
+      <div class="about-me-frameworks">
+        <img class="frameworks-icon" src="./assets/images/frameworks.svg">
+        <h3 class="frameworks-header">Frameworks</h3>
+        <ul class="frameworks-list">
+          <li class="framework-element">Bootstrap</li>
+          <li class="framework-element">Ruby on rails</li>
+          <li class="framework-element">RSpec</li>
+          <li class="framework-element">Capybara</li>
+          <li class="framework-element selenium">Selenium</li>
+        </ul>
+      </div>
+      <div class="about-me-skills">
+        <img class="skills-icon" src="./assets/images/skills.svg">
+        <h3 class="skills-header">Skills</h3>
+        <ul class="skills-list">
+          <li class="skill-element">Codekit</li>
+          <li class="skill-element">GitHub</li>
+          <li class="skill-element">Codepen</li>
+          <li class="skill-element">Gitlab</li>
+          <li class="skill-element terminal">Terminal</li>
+        </ul>
+      </div>
+    </section>
   </main>
+  <img class="about-me-first-picture" src="./assets/images/about_me_first_picture.svg">
+  <img class="about-me-second-picture" src="./assets/images/about_me_second_picture.svg">
```

Comment:
- [ ] The section has the proper background image (decorations are always backgrounded images, not `img` tags), please kindly fix it 🙃 🙃